docs/apis-tools/working-with-apis-tools.md

```diff
@@ -34,12 +34,12 @@ Camunda Platform 8 provides several official clients based on this API. Official
 
 Official clients have been developed and tested by Camunda. They also add convenience functions (e.g. thread handling for job workers) on top of the core API.
 
-<DocCardList items={[{type:"link", href:"/docs/apis-tools/java-client/", label: "Java client", docId:"apis-tools/java-client/index"},
+<DocCardList items={[{type:"link", href:"/docs/next/apis-tools/java-client/", label: "Java client", docId:"apis-tools/java-client/index"},
```

Comment:
@christinaausley I added `next/` to the href for the links in this version....it kind of stinks because the `$docs$` markdown plugin doesn't work here, since the links are part of a JavaScript object. I'll think about this some more to see if I can figure out a way for us to make these links not tied to a specific version, so that we don't have to navigate this during releases.